src/main/scala/mesosphere/marathon/api/LeaderProxyFilter.scala

```diff
@@ -9,17 +9,22 @@ import javax.servlet._
 import javax.servlet.http.{ HttpServletRequest, HttpServletResponse }
 
 import akka.Done
+import akka.http.scaladsl.model.StatusCodes._
 import com.google.inject.Inject
 import mesosphere.chaos.http.HttpConf
 import mesosphere.marathon.core.election.ElectionService
 import mesosphere.marathon.io.IO
+<<<<<<< HEAD
```

Comment:
looks like conflict markers were committed?

Reply:
that is odd..  looking